ui/base_list_ui.py

```diff
@@ -517,19 +521,19 @@ def refresh(self):
         self.o.display_image(canvas)
 
     def get_scrollbar_top_bottom(self):
-        scrollbar_max_length = self.o.height-(self.scrollbar_y_offset*2)
+        scrollbar_max_length = self.o.height - (self.scrollbar_y_offset * 2)
         total_entry_count = len(self.el.contents)
         entries_before = self.first_displayed_entry
         entries_after = total_entry_count - self.last_displayed_entry - 1
         entries_on_screen = total_entry_count - entries_before - entries_after
         if entries_on_screen >= total_entry_count:
-            #No scrollbar if there are no entries not shown on the screen
-            return (0, 0) #Look, an owl
-        #Scrollbar length per one entry
-        length_unit = float(scrollbar_max_length)/total_entry_count
-        top = self.scrollbar_y_offset + int(entries_before*length_unit)
-        length = int(entries_on_screen*length_unit)
-        bottom = top+length
+            # No scrollbar if there are no entries not shown on the screen
+            return 0, 0
```

Comment:
Oh noes no more owl

Reply:
don't be sad:   here's one for you 🦉

Reply:
This seems to be a suitable replacement; fixed